frontend/js/complex.js

```diff
@@ -512,3 +512,59 @@ $(document).ready(() => {
             });
     }
 });
+
+
+TourHelper.attachTour('#help-menu-item', [
+    {
+        target: '#spacedText',
+        content: 'This page shows you all about a specific complex.',
+        placement: ['bottom']
+    },
+    {
+        target: '#experiment-number',
+        content: 'Here you can switch between the SAME complex, found in different experiments. The experiment is listed here, if it contains at least one protein of this complex.',
+        placement: ['left', 'right', 'top', 'bottom']
+    },
+    {
+        before: () => {
+            $('#experiment-number input.search').focus();
+            return new Promise((res, rej) => {
+                setTimeout(() => {res();}, 200);
+            });
+        },
+        target: '#experiment-number > div > .menu',
+        content: 'This is a list of all the experiments, in which this complex is used.',
+        placement: ['bottom'],
+        after: () => {
+            $('#experiment-number input.search').focus();
+            return Promise.resolve();
+        }
+    },
+
+    // combined curves
```

Comment:
--> Can you select via JS the first experiment in the list? So that the graph will be populated somehow?

Reply:
Not easy, but I got it working. Here applies the same: the data of Chris has to be visible and searchable. ✅